LLaMA3.1 SFT 时出现 NaN

参考[指令微调示例](https://github.com/alibaba/Pai-Megatron-Patch/blob/main/examples/llama3_1/README.md#%E6%8C%87%E4%BB%A4%E5%BE%AE%E8%B0%83%E7%A4%BA%E4%BE%8B)执行SFT时出现NaN。其中，数据集来自[小规模预处理数据下载试用](https://github.com/alibaba/Pai-Megatron-Patch/tree/main/toolkits/sft_data_preprocessing#%E5%B0%8F%E8%A7%84%E6%A8%A1%E9%A2%84%E5%A4%84%E7%90%86%E6%95%B0%E6%8D%AE%E4%B8%8B%E8%BD%BD%E8%AF%95%E7%94%A8)。脚本为：

```
UB_SKIPMC=1 sh run_mcore_llama3_1.sh  \
dsw  \
8B   \
1    \
4 \
1e-5   \
1e-6   \
128  \
128  \
bf16  \
4   \
1  \
1 \
true \
true   \
true \
true \
false   \
false \
100000  \
/DATA/DATASETS/Pai/qwen-datasets/mmap_qwen2_sft_datasets_text_document   \
/DATA/DATASETS/Pai/qwen-datasets/mmap_qwen2_sft_datasets_text_document   \
/DATA/DATASETS/Pai/llama3-ckpts/Meta-Llama-3.1-8B/mcore-tp4-pp1  \
10000  \
100   \
/DATA/DATASETS/Pai/output_mcore_llama3_1-finetune
```
`[rank3]: AssertionError: Rank 3: found NaN in local grad norm in backward pass before data-parallel communication collective. Device: 3, node: Du`

--- a/toolkits/sft_data_preprocessing/README.md
+++ b/toolkits/sft_data_preprocessing/README.md
@@ -64,6 +64,8 @@ qwen-datasets
    └── mmap_qwen2_sft_datasets_text_document.idx
 ```
 
+需要注意的是，对于拥有不同Tokenizer的模型，按照上述流程处理得到的mmap数据不能混用，以免造成训练失败。
+
 #### Sequence Packing
 
 目前Pai-Megatron-Patch中的部分模型(LLaMA3.1, Qwen-2等)已支持基于mmap格式的Sequence-Packing训练，为此，您首先需要按照下列步骤准备打包后的数据集
@@ -97,7 +99,7 @@ LLama3Tokenizer \
 4. 微调训练时，设置`SFT=true`，同时设置环境变量`MP_SFT_PACKING=true`即可使用Sequence Packing。
 
 #### 小规模预处理数据下载试用
-为方便用户试用，我们也提供了已经处理好的小规模数据，可直接下载使用
+对于Qwen2模型，为方便用户试用，我们也提供了已经处理好的小规模数据，可直接下载使用。
 ```bash
 cd /mnt/workspace/qwen-datasets
 wget https://atp-modelzoo-wlcb-pai.oss-cn-wulanchabu.aliyuncs.com/release/models/pai-megatron-patch/qwen-datasets/mmap_qwen2_sft_datasets_text_document.bin

